youtube_dl/extractor/picarto.py

```diff
@@ -55,7 +55,7 @@ def _real_extract(self, url):
             'token': token,
         }
 
-        prefered_edge = cdn_data.get('preferedEdge')
+        preferred_edge = cdn_data.get('preferredEdge')
```

Comment:
Do not touch string literals affecting extraction.

Reply:
Reverted :smiley: